resources/zib/zib-MedicalDevice.xml

```diff
@@ -0,0 +1,180 @@
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="DeviceUseStatement.timing[x]:timingPeriod">
+      <path value="DeviceUseStatement.timing[x]" />
+      <sliceName value="timingPeriod" />
+      <type>
+        <code value="Period" />
+      </type>
+    </element>
+    <element id="DeviceUseStatement.timing[x]:timingPeriod.start">
+      <path value="DeviceUseStatement.timing[x].start" />
+      <short value="StartDate" />
+      <definition value="The start date of the first use or implant of the medical device. A ‘vague’ date, such as only the year, is permitted." />
+      <alias value="BeginDatum" />
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.11" />
+        <comment value="StartDate" />
+      </mapping>
+    </element>
+    <element id="DeviceUseStatement.timing[x]:timingPeriod.end">
+      <path value="DeviceUseStatement.timing[x].end" />
+      <short value="EndDate" />
+      <definition value="The end date of the last use or explant of the medical device. A ‘vague’ date, such as only the year, is permitted." />
+      <alias value="EindDatum" />
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.14" />
+        <comment value="EndDate" />
+      </mapping>
+    </element>
+    <element id="DeviceUseStatement.device">
+      <path value="DeviceUseStatement.device" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicalDevice-Device" />
+      </type>
+    </element>
+    <element id="DeviceUseStatement.reasonReference">
+      <path value="DeviceUseStatement.reasonReference" />
+      <short value="Indication" />
+      <definition value="The medical reason for use of the medical device." />
+      <alias value="Indicatie" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Condition" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Observation" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/DiagnosticReport" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/DocumentReference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Media" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Problem" />
+      </type>
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.7" />
+        <comment value="Indication" />
+      </mapping>
+    </element>
+    <element id="DeviceUseStatement.bodySite">

```

Comment:
Binding to http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.11--20200901000000 is required here.

Reply:
I am not sure which valueset you mean?
* MedicalDevice does not define a ValueSet for AnatomicalLocation. 
* Can't find this valueset in the zib AnatomicalLocation
* Can find this valueset in HearingFunction or VisualFunction (and Art-Decor is down)

Reply:
It's the ValueSet from zib AnatomicalLocatuion. See the profilers guidance in het zib-AnatomicalLocation datatype profile.

Reply:
Fixed!